liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -299,44 +288,183 @@ public ServerTask getServerTask() throws Exception {
             return artifactPaths;
         }
 
+        public boolean compareProperties(Properties p, Properties q) {
```

Comment:
Properties is a HashTable so I think this method could just be replaced by `p.equals(q)` or `Objects.equals(p, q)` (which handles nulls)